habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1537,7 +1536,16 @@ def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
         # Init goto velocity controller
-        self.w2v_controller = ContinuousController(self._config)
+        try:
+            from habitat.utils.controller_wrapper import ContinuousController
+
+            self.w2v_controller = ContinuousController(self._config)
+        except ModuleNotFoundError:
+            print(
+                "Missing dependencies for waypoint type actions. "
+                "Install habitat-lab with the 'continuous_control' option to enable this feature."
+            )
+            raise
```

Comment:
```suggestion
        except ModuleNotFoundError as exc:
            additional_error_message = """
            Missing dependencies for waypoint type actions. 
            Install habitat-lab with the 'continuous_control' option to enable this feature.
            pip install -e "habitat-lab[continuous_control]
            """
            exc.message += additional_error_message
            raise exc
```

Reply:
Please make sure this error is raised properly and that the message looks good. No need for a unit test but we need to make sure this works.